src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  currentTodo: Todo | null
```

Comment:
What's the purpose of rendering TodoModal if currentTodo is null?

Reply:
when there is no null rendering. Render only when not null. This is to reset currentTodo